test/functional/foreman_discovery/concerns/hosts_controller_extensions_test.rb

```diff
@@ -9,12 +9,10 @@ class HostsControllerTest < ActionController::TestCase
         test 'get "host" params from "discovered_hosts" params' do
           os = FactoryBot.create(:operatingsystem, :with_associations)
           arch_id = os.architectures.first.id
-          discovered_host_params = {
-            'discovered_host' => { 'architecture_id' => arch_id }
-          }
+          params = { 'host' => { 'architecture_id' => arch_id } }
```

Comment:
How does the page actually send the params? this should match that, so there might also be a change needed to the view?

Reply:
Oh that was a leftover when I was playing around. It's `discovered_host`, reverted.